app/controllers/api/v4/converts_controller.rb

```diff
@@ -14,7 +14,10 @@ def create
     end
 
     old_extension = Run.program(@run.program).file_extension
-    new_extension = Run.program(params[:format]).file_extension
+    if Run.program(params[:format])
+      new_extension = Run.program(params[:format]).file_extension
+    else
+      new_extension = params[:format]
```

Comment:
yeah that was a mistake, thought i ran all the tests again before i pushed but it looks like i just ran one folder that didn't include convert.  i ran them all this time for sure though